### PR TITLE
Added bokeh-based piano roll plotting

### PR DESCRIPTION
**Pull request by douglaseck**

### CODE REVIEW

File: magenta/music/notebook_utils.py [hunk 1]
```diff
@@ -16,6 +16,11 @@
 # internal imports
 import IPython
 
+import collections
+import bokeh
```
Comment (cghawthorne) [inline magenta/music/notebook_utils.py]: We should update magenta/tools/pip/setup.py to have dependencies on pandas and bokeh.

Should also add a dependency in the BUILD file with "# bokeh dep" and "# pandas dep" that we can handle internally.

Comment (douglaseck) [inline magenta/music/notebook_utils.py]: Done. Thanks!

File: magenta/music/notebook_utils.py [hunk 2]
```diff
@@ -16,6 +16,11 @@
 # internal imports
 import IPython
 
+import collections
```
Comment (cghawthorne) [inline magenta/music/notebook_utils.py]: Should change import order:

```
import collections

# internal imports

import bokeh
import bokeh.plotting
import IPython
import pandas as pd

from magenta.music import midi_synth
```

File: magenta/music/notebook_utils.py [hunk 3]
```diff
@@ -38,3 +43,72 @@ def play_sequence(sequence,
   array_of_floats = synth(sequence, sample_rate=sample_rate, **synth_args)
   IPython.display.display(IPython.display.Audio(array_of_floats,
                                                 rate=sample_rate))
+
+
+def plot_sequence(sequence,
+                  show_figure=True):
+  """Creates an interactive pianoroll for a tensorflow.magenta.NoteSequence.
```
Comment (cghawthorne) [inline magenta/music/notebook_utils.py]: Should have a blank line after the one-sentence docstring summary.

Comment (douglaseck) [inline magenta/music/notebook_utils.py]: Fixed.

**Pull request merged**